src/components/video.tsx

```diff
@@ -1,17 +1,22 @@
+/* eslint-disable react/react-in-jsx-scope */
 import { jsx } from "@emotion/core";
 import Card from "./card";
 
-function format(id: string): string {
+function formatYoutube(id: string | undefined): string {
```

Comment:
You can add ? After Id before the colons to make it optional and that way you don’t need to union with undefined

Reply:
Cool 👍 I'm using this project as a way to learn/practice typescript